packages/tsconfig/tsconfig.app.json

```diff
@@ -2,10 +2,9 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "allowJs": true,
-    // Allow spreading Maps/Sets into arrays
-    "downlevelIteration": true,
+    "noUnusedLocals": true,
```

Comment:
Small addition to match `tsconfig.package.json`. I believe this doesn't come automatically with `strict` mode. This yielded the only code change in this PR.

I did not enable `noUnusedParameters` here, there are a lot of violations and that has dubious benefit.